improver/utilities/save.py

```diff
@@ -90,7 +90,11 @@ def save_netcdf(cubelist, filename, compression_level=1):
         compression_level (int):
             1-9 to specify compression level, or 0 to not compress (default compress
             with complevel 1)
-
+        least_significant_digit (int):
+            Power of ten of the smallest decimal place in unpacked data that is a reliable value, see
```

Comment:
One of the things that's really confusing about the documentation for this is that it's a) decimal rather than significant digit and b) negative rather than positive. It'll also truncate the precision without compression on - it just improves the efficiency once it is enabled. Would you mind updating this?

Reply:
Sure :-)